src/components/InventoryHostStaleness/HostStalenessCard.js

```diff
@@ -68,13 +64,61 @@ const HostStalenessCard = () => {
         filterKey !== 'id' &&
         (apiData[filterKey] = daysToSecondsConversion(newFormValues[filterKey]))
     );
+
     //system_default means the account has no record, therefor, post for new instance of record.
     if (filter.id === 'system_default') {
-      axios.post(`${INVENTORY_API_BASE}/account/staleness`, apiData);
+      axios
+        .post(`${INVENTORY_API_BASE}/account/staleness`, apiData)
```

Comment:
Hm, why it's a patch, not post method?

Reply:
Its how the backend was designed. If the user never created an instance of staleness, the default one is applied. Once one is applied, if you try to make a post request, the api will return a failure messages because an instance already exist.